File: app/src/main/java/de/chrisward/theyworkforyou/database/LordStore.java
package de.chrisward.theyworkforyou.database;

import android.arch.lifecycle.LiveData;
import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Insert;
import android.arch.persistence.room.Query;

import java.util.List;

import de.chrisward.theyworkforyou.model.Lord;

@Dao
public interface LordStore {
    @Query("SELECT * FROM lords order by name")
    LiveData<List<Lord>> selectAllLords();

    @Insert
    void insert(Lord... products);
}

File: app/src/main/java/de/chrisward/theyworkforyou/database/LordStore.kt
package de.chrisward.theyworkforyou.database

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query

import de.chrisward.theyworkforyou.model.Lord

@Dao
interface LordStore {
    @Query("SELECT * FROM lords order by name")
    fun selectAllLords(): LiveData<List<Lord>>

    @Insert
    fun insert(vararg products: Lord)
}
